### PR TITLE
Add documentation on static asset caching

### PR DESCRIPTION
**Pull request by rschick**
Let me know if this is too simple and I should add more detail about the `X-Cache-Control` header, etc.

### CODE REVIEW

**Comment by jeremydaly** [comment]
I agree, let’s explain what’s different about the environments. The idea that we automatically handle this for you is another one of those tiny wins that developers don’t think about, then hits them in the face. Don’t be afraid to sell it!

**Comment by rschick** [comment]
Added more details, let me know what you think!

File: docs/apps/static-assets.md [hunk 1]
```diff
@@ -58,3 +58,15 @@ src
...
 
 **NOTE:** Please be sure to restart your cloud shell after changing the `.serverlessignore` file.
+
+## Static asset caching
+
+Static assets are automatically cached in Cloud's Content Delivery Network (CDN) in edge locations around the world so download speeds will be very fast.
+
+Caching works differently in "stage" and "personal" instances.
+
+In stage instances static assets are cached in the CDN for up to 24 hours. Responses will include a `Cache-Control` header that tells the CDN to cache the asset for 24 hours, and tells the browser to cache the asset and "revalidate" it before using it. When you deploy a new version of your application, Cloud will automatically clear the CDN cache so your users will get the latest version when they refresh the browser.
```
Comment (jeremydaly) [inline docs/apps/static-assets.md:68]: Need a comma after instances.

Comment (jeremydaly) [reply docs/apps/static-assets.md:68]: And "used _in_ stage instances" at the end of this paragraph

**Review by jeremydaly — CHANGES_REQUESTED** [review]

**Comment by emrahssamdan** [comment]
Let's merge this? @rschick @jeremydaly

**Pull request merged**